symbols/overlay29.yml

```diff
@@ -332,6 +341,14 @@ overlay29:
         
         r0: pointer to the status turn counter
         return: new counter value
+    - name: NextFrame
```

Comment:
I don't know if it's 100% consistent, but all functions seem to have verbs in their names, so I think we should rename this to something like `AdvanceToNextFrame`

Reply:
`AdvanceFrame` perhaps?